huggingface_pytorch-pretrained-bert_gpt.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: pytorch_hub_detail
+background-class: pytorch-hub-background
+body-class: pytorch-hub
+title: GPT
+summary: GPT models
+category: research
```

Comment:
We've renamed this label to `researchers` ;) It'd great if you can keep it consistent. Thanks!